src/en/news/blog/2008/client-cache-coherency/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Client cache coherency"
+date: "2008-03-24"
+author: "sage"
+tags: 
```

Comment:
I've noticed lots of articles don't have any tags, is that correct?

Reply:
Correct, some posts were never given one.